setup.py

```diff
@@ -67,6 +67,7 @@
             "schemas/*.xsd",
         ],
         "sbom": ["*.spdx", "*.json"],
+        "mismatch": ["*"],
```

Comment:
```suggestion
        "mismatch": ["*.py"],
```
Since we're doing file extensions with the other ones, I think we should do the same with mismatch.

Reply:
Alright! added.